### DISCUSSION TITLE
error when i run the script

### DISCUSSION THREAD

**Discussion by Draxin** [farag2/Sophia-Script-for-Windows#446] [Q&A] [answered]
![Captura](https://user-images.githubusercontent.com/60172166/215611146-6d8fe02a-b896-484a-8405-8d905dbd6249.PNG)
I translate what it says in yellow: 
WARNING: A general error has occurred that is not covered by a more specific error code.
WARNING: Some Windows components may be broken
what can i do to fix it?, I tried to run it in powershell 7 but it's the same error ( I just installed windows 10 and upgraded it this morning )

**Comment by farag2** [comment, marked as answer]
Hi. Did you remove Defender?

**Reply by Draxin** [reply]
thank you, it works
